src/main/java/com/recycle/buddy/model/Image.java

```diff
@@ -0,0 +1,23 @@
+package com.recycle.buddy.model;
+
+public class Image {
+
+    private String id;
```

Comment:
Could these fields be made final? Would there ever be a case where we would need to change their value?